getting-started/gopher-run/cmd/main.go

```diff
@@ -0,0 +1,133 @@
+package main
```

Comment:
Please add a license header (copy from the other files in this repo).

Reply:
Done